exasol/slc/internal/tasks/upload/deploy_info.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+
+import exasol.bucketfs as bfs  # type: ignore
+
+from exasol.slc.models.deploy_result import DeployResult
+from exasol.slc.models.language_definitions_builder import LanguageDefinitionsBuilder
+
+
+@dataclass
+class DeployInfo:
+    release_path: str
+    url: str
```

Comment:
do we need this still?

Reply:
It's used in line 35 in the same file.

I could remove it from here and copy the implmentation of https://github.com/exasol/script-languages-container-tool/blob/e50e8d0f73e8adf59057e2d29d49fdda1b6979d3/exasol/slc/internal/tasks/upload/deploy_container_base_task.py#L97 
to this file. What do you think?